docs/architecture/grpc-for-wcf-developers/encryption.md

```diff
@@ -0,0 +1,29 @@
+---
+title: Encryption and network security - gRPC for WCF Developers
+description: Some notes on network security and encryption in gRPC
+author: markrendle
+ms.date: 09/02/2019
+---
+
+# Encryption and network security
+
+WCF's network security model is extensive and complex, including transport-level security using HTTPS or TLS-over-TCP, and message-level security using the WS-Security specification to encrypt individual messages.
+
+gRPC massively simplifies secure networking by leaving it up to the underlying HTTP/2 implementation, which can be secured using regular TLS certificates.
```

Comment:
How is "you have to configure your certificates in Kestrel" simpler than "you have to configure your certificates with WCF"? I can configure a WCF service to use a specific certificate with a single line of code. It still needs to be done. What simplification is there?

Reply:
The simplification is that there are not as many options and configuration points in gRPC as there are in WCF. 

gRPC: HTTP/2 with TLS.

WCF: [transport/binding] using [relevant security options] [potentially with message-level encryption too].